sdk/purview/purview-scanning-rest/samples-dev/listDatasources.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how get a list of datasources
+ *
+ * @summary gets a list of datasources
+ * @azsdk-weight 40
+ */
+
+import PurviewScanning from "@azure-rest/purview-scanning";
+import { DefaultAzureCredential } from "@azure/identity";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "";
+
+async function main() {
+  console.log("== List dataSources sample ==");
+  const client = PurviewScanning(endpoint, new DefaultAzureCredential());
```

Comment:
Is emitting factories a new feature in llc code gen?

Reply:
yes a new thing for LLCs 😄 we are basing our new clients in functions and objects, no classes so far.